Tasks/VSMobileCenterUpload/typings.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Agent.Tasks.Sonoma",
+  "dependencies": {
+     "glob": "registry:npm/glob#6.0.0+20160723033700"
+  },
+  "globalDependencies": {
+    "mocha": "registry:dt/mocha#2.2.5+20160720003353",
```

Comment:
I am not sure if "typings" have the concept of devDependencies - typings are only meant for developing right?

In package.json mocha are not mentioned, I think that's where we need to separate devDepencencies.